infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/openapi/OpenApiDocBuilder.java

```diff
@@ -126,6 +143,51 @@ private void writePaths(final JsonGenerator gen) throws IOException {
     gen.writeEndObject();
   }
 
+  @SuppressWarnings("unchecked")
+  private List<Entry<String, Map<HandlerType, RestApiEndpoint>>> sortRestApiEndpoints(
+      final Map<String, Map<HandlerType, RestApiEndpoint>> endpoints) {
+    final List<Entry<String, Map<HandlerType, RestApiEndpoint>>> entries =
+        new ArrayList<>(endpoints.entrySet());
+
+    // fill desired order for tags
+    final Map<String, Integer> tagOrder = new HashMap<>();
+    final List<String> tags =
+        List.of(
+            TAG_BEACON,
+            TAG_VALIDATOR_REQUIRED,
+            TAG_VALIDATOR,
+            TAG_BUILDER,
+            TAG_REWARDS,
+            TAG_EVENTS,
+            TAG_CONFIG,
+            TAG_NODE,
+            TAG_TEKU,
+            TAG_DEBUG,
+            TAG_EXPERIMENTAL);
+    IntStream.range(0, tags.size()).forEach(i -> tagOrder.put(tags.get(i), i));
```

Comment:
It would be better to have this list coming from RestApiConstants (where tags are defined). Otherwise we'll always forget to update this when we add new tags.

Reply:
Makes sense. Done.